lib/Grape.js

```diff
@@ -25,11 +25,12 @@ class Grape extends Events {
       dht_concurrency: 24,
       dht_nodeLiveness: 5 * 60 * 1000, // 5 min
       api_port: null,
-      timeslot: 10000
+      dht_peer_maxAge: 10000
```

Comment:
@robertkowalski I think this should be 60000 as default. 10000 is a bit too low.